test/testpaint/Addresses.cpp

```diff
@@ -66,11 +67,19 @@ int32_t DISABLE_OPT RCT2_CALLPROC_X(int32_t address, int32_t _eax, int32_t _ebx,
         pop %%ebx \n\
         /* Load result with flags */ \n\
         mov %%eax, %[result] \n\
-        " : [address] "+m" (address), [eax] "+m" (_eax), [ebx] "+m" (_ebx), [ecx] "+m" (_ecx), [edx] "+m" (_edx), [esi] "+m" (_esi), [edi] "+m" (_edi), [ebp] "+m" (_ebp), [result] "+m" (result)
-        :
-        : "eax","ecx","edx","esi","edi","memory"
-    );
-    #endif
+        "
+                     : [address] "+m"(address),
+                       [eax] "+m"(_eax),
+                       [ebx] "+m"(_ebx),
+                       [ecx] "+m"(_ecx),
+                       [edx] "+m"(_edx),
+                       [esi] "+m"(_esi),
+                       [edi] "+m"(_edi),
+                       [ebp] "+m"(_ebp),
+                       [result] "+m"(result)
+                     :
+                     : "eax", "ecx", "edx", "esi", "edi", "memory");
```

Comment:
I'm not sure about this change. Is this even used?

Reply:
I think the ASM blocks need to be surrounded with `clang-format off`